lstchain/reco/r0_to_dl1.py

```diff
@@ -440,6 +445,33 @@ def r0_to_dl1(
 
                 event.r0.prefix = ''
 
+                # None values in telescope/image table
+                writer.exclude(f'telescope/image/{tel_name}', 'image_mask')
+                writer.exclude(f'telescope/image/{tel_name}', 'parameters')
+                # None values in telescope/parameters table
+                writer.exclude(f'telescope/parameters/{tel_name}', 'hadroness')
+                writer.exclude(f'telescope/parameters/{tel_name}', 'disp_norm')
+                writer.exclude(f'telescope/parameters/{tel_name}', 'disp_dx')
+                writer.exclude(f'telescope/parameters/{tel_name}', 'disp_dy')
+                writer.exclude(f'telescope/parameters/{tel_name}', 'disp_angle')
+                writer.exclude(f'telescope/parameters/{tel_name}', 'disp_sign')
+                writer.exclude(f'telescope/parameters/{tel_name}', 'disp_miss')
+                writer.exclude(f'telescope/parameters/{tel_name}', 'src_x')
+                writer.exclude(f'telescope/parameters/{tel_name}', 'src_y')
```

Comment:
I don't understand this, I thought you just wanted to exclude MC values that don't exist in real data and vice versa, to avoid warnings. But why to exclude here 'parameters' or src_x, y  both for MC and real data?

Reply:
This is correct and also what is done in the stage1 tool. `parameters` is ignored, because it is an empty container that is not being filled by lstchain (since it creates this giant `DL1Container`, not the ctapipe container structure and the rest of the columns are values that are written later not via the container system.

Reply:
Ah, I see, thanks! It is clear now.